cmds/exp/tac/tac_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2012-2023 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"errors"
+	"os"
+	"testing"
+)
+
+func TestTac(t *testing.T) {
+	f, err := os.CreateTemp("", "")
+	if err != nil {
+		t.Fatalf(`os.CreateTemp("", "") = %v, want nil`, err)
+	}
+
+	_, err = f.WriteString("hello\nworld\n")
```

Comment:
Since you used t.TempDIr, you can use WriteFIle(filepath.Join(dir, "whatever"), "hello\nworld\n", 0666)